docs/mkdocs.yml

```diff
@@ -13,8 +13,10 @@ extra:
       link: "https://github.com/developmentseed"
     - icon: "fontawesome/brands/twitter"
       link: "https://twitter.com/developmentseed"
-    - icon: "fontawesome/brands/medium"
-      link: "https://medium.com/devseed"
+    - icon: "fontawesome/brands/linkedin"
+      link: "https://www.linkedin.com/company/development-seed"
+    - icon: "fontawesome/brands/rss"
```

Comment:
@vincentsarago, silly, but I'm not sure how to validate the link that needs to be used for the font-awesome icon with doing a deployment. I found the link [here](https://fontawesome.com/icons/rss?s=solid), but it does not provide any equivalent reference to the mkdocs.

Reply:
you can run the docs locally and check if they render `mkdocs serve -f docs/mkdocs.yml` 

but right now I'm getting 👇 
```
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: .icons/fontawesome/brands/rss.svg
```

Reply:
@zacharyDez the correct rss icon link is `fontawesome/solid/rss` 

how to:

- go in https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/?h=icon#search
- type `rss`
- returns `:fontawesome-solid-rss:` 
- guess the `:fontawesome-solid-rss:` gives `fontawesome/solid/rss` 

😅

Reply:
fixed, thank you @vincentsarago !